doc/man1/flux-uri.rst

```diff
@@ -0,0 +1,144 @@
+up the URI for a Flux instance running as a job with:
+
+::
+
+   $ flux uri JOBID
+
+Depending on the *TARGET* URI scheme and corresponding plugin, specific
+query arguments int *TARGET* may be supported. However, as a convenience,
+all *TARGET* URIs support the special query arguments ``local`` or
+``remote`` to force the resulting URI into a local (``local://``) or remote
+(``ssh://``) form. For example:
+
+::
+
+   $ flux uri JOBID?local
+
+would return the ``local://`` URI for *JOBID* (if the URI can be resolved).
+
+A list of supported URI schemes will be listed at the bottom of
+``flux uri --help`` message. For a description of the URI resolver schemes
+included with Flux, see the URI SCHEMES and EXAMPLES sections below.
+
+OPTIONS
+=======
+
+**-remote**
+   Return the _remote_ (``ssh://``)  equivalent of the resolved URI.
+
+**--local**
+   Return the _local_ (``local://``) equivalent of the resulved URI.
+   Warning: the resulting URI may be invalid for the current system
+   if the network host specified by an ``ssh`` URI is not the current
+   host.
+
+URI SCHEMES
+===========
+
+The following URI schemes are included by default:
+
+jobid:ID[/ID...]
+   This scheme attempts to get the URI for a Flux instance running as a
+   job in the current enclosing instance. This is the assumed scheme if no
+   ``scheme:`` is provided in *TARGET* passed to ``flux uri``, so the
+   ``jobid:`` prefix is optional. A hierarchy of Flux jobids is supported,
+   so ``f1234/f3456`` will resolve the URI for job ``f3456`` running in
+   job ``f1234`` in the current instance.
+
+pid:PID
+  This scheme attempts to read the ``FLUX_URI`` value from the process id
+  *PID* using ``/proc/PID/environ``. If *PID* refers to a ``flux-broker``,
+  then a child of the broker is used instead in order to obtain the
+  URI for that broker (instead of its parent)
+
+slurm:JOBID
+  This scheme makes a best-effort to resolve the URI of a Flux instance
+  launched under Slurm. It invokes ``srun`` to run ``scontrol listpids``
+  on the first node of the job, and then uses the ``pid`` resolver until
+  it finds a valid ``FLUX_URI``.
+
+
+EXAMPLES
+========
+
+To get the URI of a job in the current instance in its ``local://`` form:
+
+::
+
+   $ flux uri --local ƒN8Pz2xVu
+   local:///tmp/flux-zbVtVg/jobtmp-0-ƒN8Pz2xVu/flux-59uf5w/local-0
+
+or
+
+::
+
+   $ flux uri ƒN8Pz2xVu?local
+   local:///tmp/flux-zbVtVg/jobtmp-0-ƒN8Pz2xVu/flux-59uf5w/local-0
+
+
+Get the URI of a nested job:
+
+::
+
+   $ flux uri ƒqxxTiZBM/ƒr2XFWP?local
+   local:///tmp/flux-zbVtVg/jobtmp-0-ƒqxxTiZBM/flux-EPgSwk/local-0
+
+.. note::
+   With  the ``jobid`` resolver, ``?local`` only needs to be placed on
+   the last component of the jobid "path" or hierarchy. This will resolve
+   each URI in turn as a local URI, so it is only useful if all jobs
+   are actually running on the local system.
+
+Get the URI of a local flux-broker
+
+::
+
+   $ flux uri pid:$(pidof flux-broker | cut -d' ' -f1)
```

Comment:
maybe use `pidof -s` instead of the cut?

Reply:
Cool, didn't know about `pidof -s`